Felienne/hedy
# [BUG FIX] Correctly catch invalid levels in url

**Description**
In this PR we fix a small issue where an invalid level in the url would result in a `keyErrror` instead of a nice error template. We fix this by checking `if level < 1 or level > hedy.HEDY_MAX_LEVEL`. Which should fix this error in all cases.

**Fixes**
This PR fixes #2453

**How to test**
Navigate to `http://0.0.0.0:8080/hedy/19` (for example) and verify that a nice error template is returned.
